server/__tests__/configuration.test.js

```diff
@@ -290,6 +291,84 @@ describe('Test Meilisearch plugin configurations', () => {
     expect(settings).toEqual({})
   })
 
+  test('Test configuration to remove unpublished entries', async () => {
+    const customStrapi = createFakeStrapi({
```

Comment:
Why did you chose to call it `customStrapi` instead of something like `fakeStrapi`?

Reply:
Normally i would use `customStrapi` when I add custom settings. But in this case you are right `fakeStrapi` si better